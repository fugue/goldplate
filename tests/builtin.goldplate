{
    "command": "bash",
    "input_files": "builtin/*",
    "stdin": [
        "echo GOLDPLATE_NAME: $GOLDPLATE_NAME",
        "echo GOLDPLATE_FILE: $GOLDPLATE_FILE",
        "echo GOLDPLATE_BASENAME: $GOLDPLATE_BASENAME",
        "echo GOLDPLATE_INPUT_NAME: $GOLDPLATE_INPUT_NAME",
        "echo GOLDPLATE_INPUT_FILE: $GOLDPLATE_INPUT_FILE",
        "echo GOLDPLATE_INPUT_BASENAME: $GOLDPLATE_INPUT_BASENAME"
    ],
    "asserts": [
        {"exit_code": 0},
        {"stdout": "${GOLDPLATE_NAME}.stdout"}
    ]
}
