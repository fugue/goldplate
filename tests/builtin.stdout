GOLDPLATE_NAME: builtin
GOLDPLATE_FILE: builtin.goldplate
GOLDPLATE_BASENAME: builtin.goldplate
GOLDPLATE_INPUT_NAME: builtin/1
GOLDPLATE_INPUT_FILE: builtin/1.txt
GOLDPLATE_INPUT_BASENAME: 1.txt
